runtime/doc/options.txt

```diff
@@ -2959,6 +2959,19 @@ A jump table for the options with a short description can be found at |Q_op|.
 				diff library.
 
 		algorithm:{text} Use the specified diff algorithm with the
+
+		linematch:{n}   Align and mark changes between the most
+				similar lines between the buffers. When the
+				total number of lines in the diff hunk exceeds
+				{n}, the lines will not be aligned because for
+				very large diff hunks there will be a
+				noticeable lag. A reasonable setting is
+				"linematch:60", as this will enable alignment
+				for a 2 buffer diff hunk of 30 lines each,
+				or a 3 buffer diff hunk of 20 lines each.
+
+                algorithm:{text} Use the specified diff algorithm with the
```

Comment:
It looks like there’s a copy of this text above that didn’t get deleted

Reply:
Also the indent is 16 spaces here, but it should be 2 Tabs.